packages/backend-function/src/schedule_parser.ts

```diff
@@ -1,9 +1,15 @@
-import { CronOptions, Schedule } from 'aws-cdk-lib/aws-events';
+import {
+  CronOptionsWithTimezone,
+  ScheduleExpression,
+} from 'aws-cdk-lib/aws-scheduler';
+import { TimeZone } from 'aws-cdk-lib';
```

Comment:
I see we are importing `TimeZone` here, why do we use string timezones in the rest of the changes?